runtime-data/wsl-default-settings/autobuild/overrides/etc/wsl-distribution.conf

```diff
@@ -0,0 +1,11 @@
+# https://learn.microsoft.com/windows/wsl/build-custom-distro#add-the-wsl-distribution-configuration-file
+[oobe]
+command = /usr/lib/wsl/wsl-oobe.sh
+defaultUid = 1000
+defaultName = AOSCOS
+
+[shortcut]
+icon = /usr/lib/wsl/wsl-icon.ico
+
+[windowsterminal]
+ProfileTemplate = /usr/lib/wsl/terminal-profile.json
```

Comment:
Upon second thought, these files should all be in /usr/share.

Reply:
Will fix.